src/javascript/app_2/Modules/Reports/Containers/reports.jsx

```diff
@@ -0,0 +1,56 @@
+import React          from 'react';
+import { withRouter } from 'react-router-dom';
+import VerticalTab    from 'App/Components/Elements/VerticalTabs/vertical-tab.jsx';
+import { IconClose }  from 'Assets/Settings';
+import routes         from 'Constants/routes';
+import { connect }    from 'Stores/connect';
+import { localize }   from '_common/localize';
+
+class Reports extends React.Component {
+    menu_options = () => {
+        const options = [];
+
+        this.props.routes.forEach(route => {
+            options.push({
+                default: route.default,
+                icon   : route.icon_component,
+                label  : route.title,
+                value  : route.component,
+                path   : route.path,
+            });
+        });
+
+        return options;
+    };
+
+    action_bar_items = [
+        {
+            onClick: () => { this.props.history.push(routes.trade); },
+            icon   : IconClose,
+            title  : localize('Close'),
+        },
+    ];
+
+    render() {
+        return (
+            <div className='reports'>
+                <VerticalTab
+                    header_title={localize('Reports')}
+                    action_bar={this.action_bar_items}
+                    alignment='center'
+                    classNameHeader='reports__tab-header'
+                    current_path={this.props.location.pathname}
+                    is_routed={true}
+                    is_full_width={true}
+                    list={this.menu_options()}
+                />
+            </div>
+        );
+    }
+}
+
```

Comment:
This component is missing PropTypes

Reply:
@OskarAhl the component does not make use of any props passed to it. It just uses the props assigned to it by `withRouter()` on the export statement. Is it necessary to add `PropTypes` for that?
P.S.: I have removed the `connect()` statement that was linking the `is_mobile` prop. That was just there for test and I forgot to remove it.

Reply:
I think we have `PropTypes` for all components that have props passed to them, doesn't this component make use of the props by assigning them to menu options and action_bar items?